docs/tools/util-linux.md

```diff
@@ -70,14 +70,14 @@ sdd       13.7T host-managed
 
 ## blkzone
 
-The *blkzone* command line utility allows listing (reporting) the zones of a
-zoned block device and resetting the write pointer of sequential zones. Unlike
-the *sg_rep_zone* and *sg_reset_wp* utilities of the
-[*sg3utils*](../tools/sg3utils.md) project, *blkzone* relies on the kernel
-provided ZBD `ioctl()` interface to perform zone report and zone reset
-operations. SCSI commands are not issued directly to the device by *blkzone*.
+The *blkzone* command line utility lists (reports) the zones of a zoned block
+device and resets the write pointer of sequential zones. Unlike the
+*sg_rep_zone* and *sg_reset_wp* utilities of the
+[*sg3utils*](../tools/sg3utils.md) project, *blkzone* relies on the kernel-
+provided ZBD `ioctl()` interface to perform "zone report" and "zone reset"
+operations. *blkzone* does not issue SCSI commands directly to the device.
```

Comment:
Can you change "SCSI" into "SCSI, ATA or NVMe" ? It was not there, but we should have it.